### ISSUE TITLE
[Rant] Use consistent docker container naming patterns

### ISSUE THREAD

**Issue by ccjoel** [jataware/dojo#109]
For the same corresponding docker services-

Examples of image names for 3 services:
`tasks-api`
`tasks-rqworker`
`tasks-ui`

Examples of the corresponding services in docker-compose.yaml:
`api`
`rqworker`
`ui`

Examples of container names for the same services:
`dojo-api`
`dojo-rq-worker`
`ui`

Examples of folder names:
`api`
`tasks`
`ui`

This adds cognitive overhead when performing tasks such as switch to service project directories, tail their logs, restart the service, rebuild the image, exec'ing into the container, etc.

Further examples-

tailing logs for the api:
`docker logs -f dojo-api`

tailing logs for the rqworker:
`docker logs - dojo-rq-worker`

tailing logs for the ui build dev server:
`docker logs -f ui`

It is very hard to remember when to add the `dojo` prefix, or the dashes (for `rqworker` vs `dojo-rq-worker`).
tl;dr; This has made me lose sanity and I'd like to be a sane developer again.
